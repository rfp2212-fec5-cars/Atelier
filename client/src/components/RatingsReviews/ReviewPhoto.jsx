import React, { useState } from 'react';
import Modal from '../Modal/Modal.jsx';

var ReviewPhoto = ({ photo }) => {
  const [show, setShow] = useState(false);

  return (
    <div className = 'reviewphoto'>
      <img className = 'thumbnails' src = {photo.url} onClick = {()=>setShow(true)}></img>
      <Modal onClose = {()=>setShow(false)} show = {show}>
        <img src = {photo.url}></img>
      </Modal>
    </div>

  );
};

export default ReviewPhoto;