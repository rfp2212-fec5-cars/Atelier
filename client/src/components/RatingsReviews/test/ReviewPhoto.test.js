import renderer from 'react-test-renderer';
import ReviewPhoto from '../ReviewPhoto.jsx';

it('changes the img size when clicked', ()=>{
  const componet = renderer.create(
    <ReviewPhoto photo={
      {id: 1235561, url: 'https://images.unsplash.com/photo-1515992854631-13â€¦hcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=668&q=80'}}></ReviewPhoto>
  );
  let tree = componet.toJSON();
  expect(tree).toMatchSnapshot();

});