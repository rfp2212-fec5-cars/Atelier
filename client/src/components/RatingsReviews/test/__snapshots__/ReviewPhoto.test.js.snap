// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`changes the img size when clicked 1`] = `
<div
  className="reviewphoto"
>
  <img
    className="thumbnails"
    onClick={[Function]}
    src="https://images.unsplash.com/photo-1515992854631-13â€¦hcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=668&q=80"
  />
</div>
`;
